Fortnox.NET/Models/ContractTemplates/ContractTemplate.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using FortnoxNET.Models.Invoice;
```

Comment:
I believe this is unused and can be removed.

Reply:
resolved